web/src/OverviewTable.tsx

```diff
@@ -431,6 +483,11 @@ function uiResourceToCell(r: UIResource, alertIndex: LogAlertIndex): RowValues {
   let currentBuildStartTime = res.currentBuild?.startTime ?? ""
   let isBuilding = !isZeroTime(currentBuildStartTime)
   let hasBuilt = lastBuild !== null
+  let buttons = allButtons.filter(
+    (b) =>
+      b.spec?.location?.componentType?.toLowerCase() === "resource" &&
+      b.spec?.location?.componentID === r.metadata?.name
+  )
```

Comment:
(nitpick) it'd be nice to have this as a helper function related to the custom ui buttons code, maybe like `isResourceButton`?

Reply:
yeah! though I went with the less general `buttonsForResource` (which also does the filter) since for now that's the only way we'd use it and I'm not sure if that will change.